ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
> No, Alex said dependency on core.css is implicit with any other css dependency.

@arschmitz how must builder figure out `core.css` is a dependency if it's not listed in any `js` file?